core/src/entry/mod.rs

```diff
@@ -12,6 +12,7 @@ impl CanPublish for EntryType {
         match self {
             EntryType::Dna => return false,
             EntryType::CapTokenGrant => return false,
+            EntryType::CapTokenClaim => return false,
```

Comment:
This could become `EntryType::CapTokenGrant | EntryType::CapTokenClaim => return false`. In a future PR we should isolate any use of a wildcard when pattern matching and explode out the enum cases.

Reply:
yep, updated